monkestation/code/modules/mob/living/carbon/human/monkey/monkey.dm

```diff
@@ -0,0 +1,34 @@
+/mob/living/carbon/human/species/monkey/ook
+	name = "Ook"
+	unique_name = FALSE
+	use_random_name = FALSE
+	ai_controller = /datum/ai_controller/monkey/ook
+	var/ancestor_name
+	var/ancestor_chain = 1
+	var/relic_hat
```

Comment:
im confused on these vars as they never have a use, and subtyping an ook monkey seems weird, also since its ook why doesn't the default path send to the ook mask?

Reply:
i'd recommend dropping these vars entirely, also removing the memory saved section as ook doesn't have persistant memory in this pr